examples/complete/firestore/README.md

```diff
@@ -66,7 +66,7 @@ Your app is ready to be deployed!
 
 ### Integrating Auth
 
-Checkout [the auth recipes](/docs/recipes/auth) for some simple examples of how to integrate auth.
```

Comment:
This link actually already works for me in both the gitbook and markdown versions of the docs. Does it not work for you? If not, how are you viewing?

Reply:
No, not working for me. These are in the Readme of the examples.

Reply:
![broken-links](https://user-images.githubusercontent.com/26483075/56332676-ebd02000-61ae-11e9-9692-7e1d0f4accaa.gif)

Reply:
Weird - Thanks for the screen capture.

You may already know this, but the gitbook rendered version of the docs is available at [react-redux-firebase.com](https://react-redux-firebase.com).

Reply:
These links are in the Readme of the complete examples- simple & firestore.